modules/darwin/community-builder/users.nix

```diff
@@ -357,7 +357,5 @@ in
 
   users.knownUsers = builtins.map (u: u.name) users;
 
-  users.forceRecreate = true;
```

Comment:
Since the upstream change from dscl to sysadminctl this option now also deletes the users $HOME>